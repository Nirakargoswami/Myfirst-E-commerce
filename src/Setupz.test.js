import React from "react";
import { shallow } from "enzyme";
import Coomponent from "./Newcomponent"

it("expect that react encyme shuld work" ,()  =>{
return expect(shallow(<Coomponent/>)).toMatchSnapshot();
} )