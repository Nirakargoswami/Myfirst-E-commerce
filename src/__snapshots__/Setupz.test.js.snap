// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect that react encyme shuld work 1`] = `ShallowWrapper {}`;
